CHANGELOG.md

```diff
@@ -49,12 +49,12 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 ## API changes
 
 #### I/O
- * Changed default of `output_options::fasta_blank_before_id` to `false`
+ * Changed the default of `output_options::fasta_blank_before_id` to `false`
    ([\#2769](https://github.com/seqan/seqan3/pull/2769)).
- * Changed default of `bgzf_thread_count` to `4` instead of determining the numbers of cores of the
-   machine it is running on ([\#2911](https://github.com/seqan/seqan3/pull/2911)).
- * `bgzf_thread_count` is now a single global variable instead of having an instance per translation unit
-   ([\#2752](https://github.com/seqan/seqan3/pull/2752)).
+ * Changed the default of `bgzf_thread_count` to `4`. Previously, all available threads on the machine were utilised
+   ([\#2911](https://github.com/seqan/seqan3/pull/2911)).
+ * The behaviour of `bgzf_thread_count` has been adapted to affect the runtime of a program. Formerly, this variable was
```

Comment:
```suggestion
 * The behaviour of `bgzf_thread_count` has been adapted to take effect during the whole runtime of a program. Formerly, this variable was
```

On re-reading, this might be clearer? `affect the runtime` sounds like it's faster/slower.
Will need a line break somewhere ...

Reply:
I think this explanation is more abstract that it has to be? Why don't we explitily say that we removed the static keyword?
```suggestion
 * The global variable `bgzf_thread_count` is not static anymore. Modifying the variable now takes effect during the runtime of the entire program, while formerly, this variable was
```